lib/elixir/pages/Typespecs.md

```diff
@@ -35,9 +35,9 @@ The syntax Elixir provides for type specifications is similar to [the one in Erl
                                     ## Numbers
```

Comment:
I think the comments here are not useful. I would rather remove `float`, `integer` and the newly added negative integer, and friends.